File: Flutter/lib/model/Member.dart
import 'dart:convert';

class Member {
  int? id;
  final String email;
  final String? password;
  String? name;
  String? phoneNumber;
  int? point;
  int? stamp;
  String? img;

  Member(
      {this.id,
      required this.email,
      this.password,
      this.name,
      this.phoneNumber,
      this.point,
      this.stamp,
      this.img});

  factory Member.fromJson(Map<String, dynamic> json) {
    return Member(
        id: json['id'],
        email: json['email'],
        name: json['name'],
        phoneNumber: json['phoneNumber'],
        point: json['point'],
        stamp: json['stamp'],
        img: json['img']);
  }

  Map<String, dynamic> toJson() => {
        'email': email,
        'password': password,
        'phoneNumber': phoneNumber,
        'name': name
      };
}

File: Back/src/main/kotlin/com/ssafy/tp/model/Member.kt
package com.ssafy.tp.model

import javax.persistence.*
import javax.validation.constraints.NotBlank

@Entity(name = "member")
data class Member(

    @Id
    @Column(name="id")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,

    @get: NotBlank
    @Column(name = "email")
    var email:String = "",

    @get: NotBlank
    @Column(name = "password")
    var password:String ="",

    @get: NotBlank
    @Column(name = "name")
    var name:String? = "",

    @Column(name = "point")
    var point:Int?,

    @Column(name = "phone_number")
    var phoneNumber:String?,

    @Column(name = "stamp")
    var stamp: Int?,

    @Column(name ="img")
    var img: String?,

) {


}